### PR TITLE
Docs: update the current documentation on notification policies

### PR DESCRIPTION
**Pull request by gillesdemey**
Since we've merged https://github.com/grafana/grafana/pull/61952 we've had to update the documentation to be correct.

Additionally I've added a task on "how to search for notification policies" and I've removed some outdated screenshots (with Brenda's blessing).

This PR does not include any planned changes to the structure of the docs.

Most notable changes in this PR are:

1. Root policy is now Default policy
1. No more specific routing
1. Add policy is now new nested policy
1. Mute timings is now under a tab
1. Remove screenshot(s)
1. Search task for notification policies

### CODE REVIEW

File: docs/sources/alerting/manage-notifications/mute-timings.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ The following table highlights the key differences between mute timings and sile
 1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page listing existing alerts.
 1. Click **Notification policies**.
 1. From the **Alertmanager** dropdown, select an external Alertmanager. By default, the Grafana Alertmanager is selected.
-1. At the bottom of the page there will be a section titled **Mute timings**. Click the **Add mute timing** button.
+1. Navigate to the tab titled **Mute Timings** by clicking it.
+1. Click the **Add mute timing** button.
```
Comment (Eve832) [inline docs/sources/alerting/manage-notifications/mute-timings.md:40]:
```suggestion
1. Click the **Mute Timings** tab.
1. Click **Add mute timing**.
```

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ Now that Grafana knows how to notify us, it's time to set up an alert rule:
 1. In **Section 4**, you can add some sample text to your summary message. [Read more about message templating here](/docs/grafana/latest/alerting/unified-alerting/message-templating/).
 1. Click **Save and exit** at the top of the page.
 1. In Grafana's sidebar, hover the cursor over the **Alerting** (bell) icon and then click **Notification policies**.
-1. Under **Root policy**, press **Edit** and change the **Default contact point** to **RequestBin**. As a system grows, admins can use the **Notification policies** setting to organize and match alert rules to specific contact points.
+1. Under **Default policy**, press **...** &rsaquo; **Edit** and change the **Default contact point** to **RequestBin**. As a system grows, admins can use the **Notification policies** setting to organize and match alert rules to specific contact points.
```
Comment (Eve832) [inline docs/sources/tutorials/grafana-fundamentals/index.md:326]:
```suggestion
1. Under **Default policy**, select **...** &rsaquo; **Edit** and change the **Default contact point** to **RequestBin**. 
     As a system grows, admins can use the **Notification policies** setting to organize and match alert rules to 
     specific contact points.
```

Comment (Eve832) [reply docs/sources/tutorials/grafana-fundamentals/index.md:326]: - Check to make sure this renders the way you expect it to: `**...** &rsaquo; **Edit**`
- Not sure if my indentation suggestion above will be formatted correctly if you commit through the UI, but it's just a nit anyway.

**Review by Eve832 — APPROVED** [review]
Mostly just NITs here; thank you @gillesdemey !

**Review by brendamuir — APPROVED** [review]

File: docs/sources/tutorials/grafana-fundamentals/index.md (hunk 2 above)
Comment (gillesdemey) [reply docs/sources/tutorials/grafana-fundamentals/index.md:326]: Yep, I've checked this with `make docs` on my local machine and renders the correct character 👍

**Pull request merged**